pay-queue/src/pay_queue/services/eft/eft_reconciliation.py

```diff
@@ -78,7 +65,7 @@ async def reconcile_eft_payments(msg: Dict[str, any]):  # pylint: disable=too-ma
         EFTFileModel.file_ref == file_name).one_or_none()
 
     if eft_file_model and eft_file_model.status_code == EFTProcessStatus.COMPLETED.value:
```

Comment:
Might need to do the same thing here, have a state where it's started processing at the very beginning. Same with CSV.

Reply:
The eft_file record is created or updated just below this and then flushed/committed so it should show up on any other db sessions.

We can probably just update to also check for EFTProcessStatus.IN_PROGRESS in addition to COMPLETED.

Reply:
Working on this now